README.md

```diff
@@ -16,3 +16,22 @@ To enable external control of a UR robot from a remote PC, this URcap must be in
 	* Add this URcap to a program by selecting it from the side menu under the tab _URcap_.
 	* Execute the program by pressing the _play_ button in the _Program_ tab of Polyscope.
 
+### Multiple URCap nodes
+To use this URCap node multiple times in a ur program, the control script is divided into two
+scripts. After receiving the script, it is divided into a header part and a control loop part. The
+header part consist of all the function deffinitions. The header is only inserted once in the 
+program, while the control loop is inserted for each URCap node in the program tree.
+
+To be able to distinguish between header and control loop, the header part of the script should be
+encapsulated in:
+```bash
+# HEADER_BEGIN
+Here goes the header code
+# HEADER_END
+
+# NODE_CONTROL_LOOP_BEGINS
+Here goes the control loop code
+# NODE_CONTROL_LOOP_ENDS
```

Comment:
```suggestion
Here goes the control loop code
```
These anchors actually aren't used in the implementation. Which I think is good, as this would raise the question, what would happen with any code not inside any anchors.

However, I think we should remove it from the documentation, then.